File: 268-missing-number/missing-number.java
class Solution {
    public int missingNumber(int[] nums) {
        int n=nums.length;
        int output=n*(n+1)/2;
        int act_sum=0;
        for(int i:nums){
            act_sum+=i;
        }
        return output-act_sum;
    }
}

File: 268-missing-number/missing-number.py
class Solution:
    def missingNumber(self, nums: List[int]) -> int:
        nums=sorted(nums)
        res=0
        val=nums[0]
        for i in range(len(nums)):
            if(val+1) in nums:
                val=val+1
            else:
                res=val+1
        if(nums[0]!=0):
            return 0
        else:
            return res

        